plugin.program.bluetoothctl/addon.py

```diff
@@ -0,0 +1,386 @@
+from functools import wraps
```

Comment:
It's not good form to have the entire program logic loaded by the file called from the addon.xml (in this case addon.py).  That file in never cached, so there is a significant performance hit.  Instead, import from a different file and just call it from addon.py.